config.go

```diff
@@ -438,6 +445,18 @@ func (s *Settings) GetGithubOauthToken() (string, error) {
 	return "", errors.New("no github token in settings")
 }
 
+func (s *Settings) SetUpNewRelic() (*newrelic.Application, error) {
```

Comment:
can you make this a method on the NewRelicConfig type

Reply:
done